### DISCUSSION TITLE
Support AVIF file format

### DISCUSSION THREAD

**Discussion by pablorosatti** [sumatrapdfreader/sumatrapdf#3286] [Ideas]
I have converted my collection of Comic Books CBR (JPEG images compressed with RAR) to CB7 (WEBP images compressed with 7zip) managing to reduce its size drastically without losing quality. I just did image tests with the AVIF image format (also from Google, like WEBP) and I managed to reduce the size even more (almost to half of WEBP) without reducing the image quality, the only problem is that SumatraPDF does not recognize the AVIF image format.
I leave a link for you to compare the 3 files that I used in the test:
1- Original (JPG) 1802.24k
2- WEBP compressed with ImageMagick (quality 17) 265k
3- AVIF compressed with ImageMagick (quality 25) 156k

[Test.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/10775119/Test.zip)

I propose that SumatraPDF add support for the AVIF image format.

**Comment by kjk** [comment]
AVIF is already supported in latest pre-release version: https://www.sumatrapdfreader.org/prerelease
